pyscriptjs/src/interpreter.ts

```diff
@@ -39,20 +39,39 @@ const loadPackage = async function (package_name: string[] | string, runtime: an
 
 const loadFromFile = async function (s: string, runtime: any): Promise<any> {
     const filename = getLastPath(s);
-    await runtime.runPythonAsync(
-        `
-        from pyodide.http import pyfetch
-        from js import console
-        response = await pyfetch("` +
-            s +
-            `")
-        content = await response.bytes()
-        with open("` +
-            filename +
-            `", "wb") as f:
-            f.write(content)
-    `,
-    );
+    try {
+        await runtime.runPythonAsync(
+            `
+            from pyodide.http import pyfetch
+            from js import console
+
+            response = None;
+            try:
+                response = await pyfetch("` +
+                    s +
+                    `")
+            except Exception as err:
+                console.warn("PyScript: Access to local files (using 'Paths:' in py-env) is not available when directly opening a HTML file; you must use a webserver to serve the additional files. See https://github.com/pyscript/pyscript/issues/257#issuecomment-1119595062 on starting a simple webserver with Python.")
+            content = await response.bytes()
```

Comment:
are you sure this works as expected? If pyfetch raises before assigning response, it'll be None so will not have a `.bytes` method.... or am I missing something?

Reply:
You're exactly right - the try/except block here was swallowing the error from pyfetch, which made the AttributeError from calling `None.bytes()` be the error that was actually raised.

I've modified loadFromFile() so sends the same 'PyScript: Access to local files' error to the console, then immediately raises the error from pyfetch. Thus, the user sees the 'Failed the Fetch' error in the console, which is probably better than the AttributeError.

```
from pyodide.http import pyfetch
from js import console

try:
    response = await pyfetch("` +
        s +
        `")
except Exception as err:
    console.warn("PyScript: Access to local files (using 'Paths:' in py-env) is not available when directly opening a HTML file; you must use a webserver to serve the additional files. See https://github.com/pyscript/pyscript/issues/257#issuecomment-1119595062 on starting a simple webserver with Python.")
    raise(err)
content = await response.bytes()
...
```
![image](https://user-images.githubusercontent.com/1931111/168088608-7c8b0eaa-c390-4573-a62e-293317a5a2f7.png)